app/graphql/types/query_type.rb

```diff
@@ -20,12 +20,27 @@ def nodes(ids:)
 
     # Add root-level fields here.
     # They will be entry points for queries on your schema.
+    
+    field :monsters, [Types::MonsterType], null: false,
+      description: "A complete list of Monsters" 
+      def monsters
+        MonsterFacade.monster_list
+      end
 
-    # TODO: remove me
-    field :test_field, String, null: false,
-      description: "An example field added by the generator"
-    def test_field
-      "Hello World!"
-    end
+    field :monster, [Types::MonsterType], null: false,
+      description: "Details for one monster" do
+        argument :index, String
+      end
+      def monster(index:)
+        MonsterFacade.single_monster_details(index)
+      end
+
+    field :encounters, [Types::EncounterType], null: false,
+      description: "A complete list of a users encounters" do
+        argument :userName, String, required: true 
+      end
+      def encounters(userName:)
+        encounters = Encounter.where(user_name: userName)
+      end
```

Comment:
Similar to my note before, I think we should keep this as is, but for future.

We can narrow down the data sent off in the endpoint for this as a future refactor. It honestly is good to keep as is, just in case frontend needs any additional information to display on the `index` page. and they can just simply query less data when sending the request.

Reply:
I'm not fully sure what you mean here, what data would we be pairing down, and how would we accomplish that? (this might be a better discussion over zoom/slack)

Reply:
For this `index` page, frontend is only displaying encounterName, summary, encounterMonsters, partySize, and partyLevel.

This endpoint, how it is set up, contains additional data that is used in the FULL encounter `show` page (treasure, etc.)

Again, not a big issue, as they can filter what is queried, based on the query body they send in a request. Just marking it as a potential cleanup item in the future!